README.md

```diff
@@ -16,6 +16,21 @@ cargo build --features stm32f103
 
 If no device is specified, the crate does not compile.
 
+Alternatively, adding the following to your project's `Cargo.toml` will target
+your chip without needing the `--features` argument:
+
+```
+[features]
```

Comment:
The same can be achieved more easily by simply enabling the features in the dependency section:
e.g.
```
[dependencies.stm32f1xx-hal]
features = ["stm32f100", "rt"]
```

No need to declare new features in the using crate and forward them.

Reply:
Yeah, that's a far better idea. Updated.